buildSrc/crosslibs/crosslibs-armv6hf.sh

```diff
@@ -228,7 +228,7 @@ installLibs() {
 
     getPackages  \
         $DESTINATION \
-        http://ftp.us.debian.org/debian/ wheezy main armhf \
+        http://archive.debian.org/debian/ wheezy main armhf \
```

Comment:
The use of `http://` URLs to download artifacts is strongly discouraged, since it isn't secure. Is there a valid `https://` URL that can be used instead? I note that just substituting `http` with `https` in the above URL does not work.

Reply:
In general a lot of Debian package URLs are not HTTPS by default because of apt's built-in signature checking, https://whydoesaptnotusehttps.com/. However, it does seem like it should at least be supported, so it might be a bug in the `debian.org` server config.

Additionally, I see that the `getPackages` command doesn't check these signatures. It probably should, but that's another PR.